redis/tests/test_basic.rs

```diff
@@ -667,13 +670,50 @@ fn test_pubsub() {
     assert_eq!(con.publish("foo", 23), Ok(1));
 
     thread.join().expect("Something went wrong");
+    if ctx.protocol == ProtocolVersion::RESP3 {
```

Comment:
you can use this to remove the opt-in in the test
https://github.com/redis-rs/redis-rs/pull/1096/commits/62fe8a4e8141f8a668d47bf52faec0f8f0112842

Reply:
It's nice to know about it, but I don't want to override the original test, it should run for both RESP3 and RESP2